src/builder.rs

```diff
@@ -322,6 +328,8 @@ pub struct Builder {
     anchor: Anchor,
 }
 
+type UnauthorizedBundle<V, D> = Bundle<InProgress<Unproven<D>, Unauthorized>, V, D>;
```

Comment:
From @PaulLaux:
Nice, let's go with
```
/// A type alias for a bundle in an unproven, unauthorized,  state
pub type UnprovenUnauthorized<V, D> = Bundle<InProgress<Unproven<D>, Unauthorized>, V, D>;
```

Reply:
Done.

Reply:
I don't see it.

Reply:
https://github.com/QED-it/orchard/pull/104/files

Reply:
Done (zcash/orchard 0.8.0 introduced UnauthorizedBundle type, so as we discussed, let's keep UnauthorizedBundle name)